### DISCUSSION TITLE
指令碼錯誤 共4個

### DISCUSSION THREAD

**Discussion by Liteya** [GpointChen/FFXIVChnTextPatch-GP#456] [General]
1. [resource/rawexd/quest/048/KinGba231_04827.csv](https://github.com/GpointChen/FFXIVChnTextPatch-GP/blob/master/resource/rawexd/quest/048/KinGba231_04827.csv) (職能治療96任務)

原始碼
`75,TEXT_KINGBA231_04827_LOASHKANA_000_032,<hex:022C0DFF07><hex:022903EB0203><hex:FF02><hex:0203>、<hex:02100103>離開湯池一點點。★
114,TEXT_KINGBA231_04827_LOASHKANA_000_071,<hex:022C0DFF07><hex:022903EB0203><hex:FF02><hex:0203>那邊情況如何？★
130,TEXT_KINGBA231_04827_LOASHKANA_000_086,<hex:022C0DFF07><hex:022903EB0203><hex:FF02><hex:0203>、<hex:02100103>因為有你在，我們才得以在溫泉裡幫助大家。<hex:02100103>感謝你的陪伴，真的非常感謝。
`
建議修正
`75,TEXT_KINGBA231_04827_LOASHKANA_000_032,<hex:022C0DFF07><hex:022903EB0203><hex:FF02> <hex:0203>、<hex:02100103>離開湯池一點點。★
114,TEXT_KINGBA231_04827_LOASHKANA_000_071,<hex:022C0DFF07><hex:022903EB0203><hex:FF02> <hex:0203>那邊情況如何？★
130,TEXT_KINGBA231_04827_LOASHKANA_000_086,<hex:022C0DFF07><hex:022903EB0203><hex:FF02> <hex:0203>、<hex:02100103>因為有你在，我們才得以在溫泉裡幫助大家。<hex:02100103>感謝你的陪伴，真的非常感謝。★
`
少一個空格

2. [resource\rawexd\quest\048\KinGba251_04829.csv](https://github.com/GpointChen/FFXIVChnTextPatch-GP/blob/master/resource/rawexd/quest/048/KinGba251_04829.csv) (職能治療100任務)

原始碼
`130,TEXT_KINGBA251_04829_LOASHKANA_000_087,我回來了，尼皮涅！<hex:02100103><hex:022C0DFF07><hex:022903EB0203><hex:FF02>多虧了<hex:0203>你的幫助，我們終於成功了。★`
建議修正
`130,TEXT_KINGBA251_04829_LOASHKANA_000_087,我回來了，尼皮涅！<hex:02100103><hex:022C0DFF07><hex:022903EB0203><hex:FF02> <hex:0203>多虧了你的幫助，我們終於成功了。★`

指令錯誤 玩家名字加粗用的東西

3. [resource\rawexd\quest\048\KinGba341_04834.csv](https://github.com/GpointChen/FFXIVChnTextPatch-GP/blob/master/resource/rawexd/quest/048/KinGba341_04834.csv)  (職能近戰98任務)

原始碼
`123,TEXT_KINGBA341_04834_YAMABUKI_000_060,你前往紅玉海的「沖之岩」洞穴。<hex:02100103>放下<hex:022811FF0A>EventItem<F61E92240103>，引誘出<hex:02280FFF09>BNpcName<hex:F234990103>，如果成功擊敗它，就能獲得<hex:022811FF0A>EventItem<hex:F61E92250103>。★`
建議修正
`123,TEXT_KINGBA341_04834_YAMABUKI_000_060,你前往紅玉海的「沖之岩」洞穴。<hex:02100103><hex:022811FF0A>EventItem<hex:F61E92240103>撒下，引誘出<hex:02280FFF09>BNpcName<hex:F234990103><hex:02100103>如果成功擊敗它的話<hex:022811FF0A>EventItem<hex:F61E92250103>就能獲得。`

雖然不知道為什麼 不過是這邊有問題 

4. [resource\rawexd\quest\048\KinGbb211_04856.csv](https://github.com/GpointChen/FFXIVChnTextPatch-GP/blob/master/resource/rawexd/quest/048/KinGbb211_04856.csv) 畫家83職能任務

原始碼
`4,TEXT_KINGBB211_04856_SEQ_04,找到了哈特獸。前往庫爾札中央高地的聖丹斐安之淚前，討伐名為「<hex:02280FFF09>BNpcName<hex:F2349E0103>」的敵人，獲得「<hex:022811FF0A>特殊物品<hex:F61E922B0103>」。★`
建議修正
`4,TEXT_KINGBB211_04856_SEQ_04,找到了哈特獸。前往庫爾札中央高地的聖丹斐安之淚前，討伐名為「<hex:02280FFF09>BNpcName<hex:F2349E0103>」的敵人，獲得「<hex:022811FF0A>EventItem<hex:F61E922B0103>」。★`

關鍵詞錯誤

**Comment by Liteya** [comment]
另外別的部分是 機翻器不想工作w
https://github.com/GpointChen/FFXIVChnTextPatch-GP/blob/master/resource/rawexd/quest/048/KinGba411_04837.csv


```
49,TEXT_KINGBA411_04837_CEETOLJA_000_001,我現在只是遞解詞彙提供翻譯，無論文本的進行模式如何，我無法做到動態地回應。感謝您的理解。★

49,TEXT_KINGBA411_04837_CEETOLJA_000_001,それを教えてくれた闘士さんに声かけてくるから、<hex:02100103>ちょっと待っててー！

49,TEXT_KINGBA411_04837_CEETOLJA_000_001,我去找告訴我這件事的鬥士<hex:02100103>稍等一下★
```

**Comment by GpointChen** [comment]
感謝

[resource/rawexd/quest/048/KinGba231_04827.csv](https://github.com/GpointChen/FFXIVChnTextPatch-GP/blob/master/resource/rawexd/quest/048/KinGba231_04827.csv) (職能治療96任務)

似乎貼到了第四點的文字？

**Reply by GpointChen** [reply]
另外第三點看起來是有個框少了`hex:`，當初沒注意到

**Reply by GpointChen** [reply]
[KinGba231_04827.csv](https://github.com/GpointChen/FFXIVChnTextPatch-GP/blob/master/resource/rawexd/quest/048/KinGba231_04827.csv) 的問題也找到了，指令碼內少一個空格，感謝回報。

**Reply by Liteya** [reply]
對..是少一個空格 抱歉 我沒貼好
